migrations/V37__add_user_to_kpi_definition.sql

```diff
@@ -0,0 +1,73 @@
+ALTER TABLE public.kpi_definition
+ADD created_by uuid not null default auth.uid ();
```

Comment:
There is a slight problem with that column being not null - we wouldn't be able to add users easily directly in the database.

Reply:
yes, you're right. I now set a default value and then replace it with auth.uid():
ALTER TABLE public.kpi_definition
 ADD created_by uuid not null default '2eca1eb1-2876-4625-ab7b-ce5342343f08';

 ALTER TABLE public.kpi_definition
 ALTER COLUMN created_by SET DEFAULT auth.uid ();